components/financial_assistance/db/seedfiles/translations/en/me/financial_assistance.rb

```diff
@@ -342,7 +342,11 @@
   "en.faa.review.other_questions" => "Other Questions",
   "en.faa.review.coverage.caretaker" => "Which member(s) of the household is this person the caretaker for?",
   "en.faa.review.your_household" => "Your Household",
+  "en.faa.review.your_household.relationship_lanel" => "Relationship",
+  "en.faa.review.your_household.is" => "is",
+  "en.faa.review.your_household.of" => "of",
   "en.faa.review.your_household.relationship" => "%{related_name} is %{relationship} of",
```

Comment:
can we remove the new "is" and "of" strings in favor of the `en.faa.review.your_household.relationship`?